probe/main.go

```diff
@@ -55,129 +54,152 @@ func main() {
 		return
 	}
 
+	defer log.Print("bye!")
```

Comment:
Would be nice to make it clear this is the probe exiting: logs are mixed with app when running under docker.